File: src/main/Java/org/team1085/robot/commands/ArcadeDrive.java
package org.team1085.robot.commands;

import edu.wpi.first.wpilibj.command.Command;

import org.team1085.robot.Map;
import org.team1085.robot.OI;
import org.team1085.robot.Robot;

public class ArcadeDrive extends Command {

    public ArcadeDrive() {
        requires(Robot.chassis);
    }

    protected boolean isFinished() {
        return false;
    }

    public void execute() {
        Robot.chassis.setLeftSpeed(OI.controller1.getRawAxis(Map.L_Y_AXIS));
        Robot.chassis.setRightSpeed(OI.controller1.getRawAxis(Map.R_Y_AXIS));
    }
}

File: src/main/kotlin/org/team1085/robot/commands/ArcadeDrive.kt
package org.team1085.robot.commands

import edu.wpi.first.wpilibj.command.Command
import org.team1085.robot.Robot

class ArcadeDrive : Command () {

  init { requires(Robot.chassis) }

  override fun isFinished () = false
  override fun execute () {
    val x = Robot.dashboard.leftStick.getX()
    val y = Robot.dashboard.leftStick.getY()

    Robot.chassis.setLeftSpeed(Math.pow(y - x, 3.0))
    Robot.chassis.setRightSpeed(Math.pow(y + x, 3.0))
  }
}